docs/sdk/guides/04-dispatch-remote-substrate-call.md

```diff
@@ -121,5 +74,8 @@ Deploy the Dapp contract on the Pangoro Smart Chain. Inject the endpoint address
 
 ## Run
 
-1. You can get a estimated fee by calling [the `fee` function](../api-reference#fee) of the endpoint contract.
-2. Call the `remoteRemark(_remark)` with a value as the market fee. The value should greater than or equal to the estimated fee.
+1. You can get a estimated market fee by calling [the `fee` function](../api-reference#fee) of the endpoint contract.
+2. Call the `remoteRemark(_remark)`.
+
+   * The param `_remark` is the content you want to remark.  
+   * The value input should be set to a market fee >= the estimated market fee. 
```

Comment:
```suggestion
   * The value input should be set to a market fee(Get in the first step). 
```

Reply:
above this there is a line:

1. You can get a estimated market fee by calling the [`fee`](../api-reference#fee) function of the endpoint contract.